src/components/Pricing/PricingCalculator/Tabbed.tsx

```diff
@@ -277,11 +277,61 @@ const Addon = ({ type, name, description, plans, addons, setAddons, volume, incl
     )
 }
 
-const TabContent = ({ activeProduct, addons, setAddons }) => {
+const TabContent = ({ activeProduct, addons, setAddons, totalPrice }) => {
+    const [showMessage, setShowMessage] = useState(false)
+    const [modalOpen, setModalOpen] = useState(false)
     const { slider, calcVolume, denomination, calcCost, volume } = activeProduct
 
+    useEffect(() => {
+        if (!showMessage) {
+            setShowMessage(totalPrice >= 100)
+        }
+    }, [totalPrice])
+
     return (
         <>
+            <SideModal open={modalOpen} setOpen={setModalOpen}>
+                <h3 className="mb-2">Save money if you don't need PII</h3>
```

Comment:
We shouldn't use the term PII here as it has a specific connotation for privacy/security regulations and we mean something different. I think the key feature here is merges in terms of helping people qualify/disqualify this feature as buyers. 

```suggestion
                <h3 className="mb-2">Save money if you don't need to merge user identities across time</h3>
                 <p className="font-semibold opacity-70 text-[15px]">Merging users across cookies/devices/codebases is more expensive for us to process.</p>
                <p>For most use cases with anonymous traffic (eg traditional Web Analytics, landing pages, Ecommerce), it costs us less to process events, so we pass that savings along to you.</p>
                <p>If you are a traditional logged-in or b2b SaaS product, you probably need Person Profiles</p>
```

Reply:
Fair, but I also think "merge user identities across time" is a bit high-brow, especially at this stage. (Unless you think this is a commonly-understood term from web analytics buyers? I don't have that context - just a hunch.)

Save money if you don't need to... track individual users(?)